### PR TITLE
Added asset filter

### PR DESCRIPTION
**Pull request by jaxwilko**
This PR adds an asset filter to twig.

```twig
{{ 'path/to/asset.png' | asset }}
// this is the equivilant to the blade
{{ asset('path/to/asset.png') }}
```

<a href="https://gitpod.io/#https://github.com/wintercms/winter/pull/428"></a>

### CODE REVIEW

**Comment by jaxwilko** [comment]
> Note: This PR now additionally includes mirroring system filters as functions

This change introduces the following functionality within twig templates:

- app - `{{ 'path/to/link' | app }}` or `{{ app('path/to/link') }}`
- media - `{{ 'path/to/file.png' | media }}` or `{{ media('path/to/file.png') }}`
- asset - `{{ 'path/to/file.png' | asset }}` or `{{ asset('path/to/file.png') }}`
- resize - `{{ 'path/to/file.png' | resize }}` or `{{ resize('path/to/file.png') }}`
- imageWidth - `{{ 'path/to/file.png' | imageWidth }}` or `{{ imageWidth('path/to/file.png') }}`
- imageHeight - `{{ 'path/to/file.png' | imageHeight }}` or `{{ imageHeight('path/to/file.png') }}`

File: modules/system/twig/Extension.php [hunk 1]
```diff
@@ -55,6 +63,7 @@ public function getFilters()
         $filters = [
             new TwigSimpleFilter('app', [$this, 'appFilter'], ['is_safe' => ['html']]),
             new TwigSimpleFilter('media', [$this, 'mediaFilter'], ['is_safe' => ['html']]),
+            new TwigSimpleFilter('asset', [$this, 'assetFilter'], ['is_safe' => ['html']]),
```
Comment (LukeTowers) [inline modules/system/twig/Extension.php]: Can you remove the is_safe flags from the filters as well so that the function and filter behaviour matches?

Comment (jaxwilko) [reply modules/system/twig/Extension.php]: Done in https://github.com/wintercms/winter/pull/428/commits/83854069beca193a0beb1790a50bb9eef5aad001 :)

File: modules/system/twig/Extension.php [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <?php namespace System\Twig;
 
+use Twig\TwigFunction as TwigSimpleFunction;
```
Comment (LukeTowers) [inline modules/system/twig/Extension.php]: Please put this next to the other twig import

Comment (jaxwilko) [reply modules/system/twig/Extension.php]: My bad, it was auto imported at the top lol. Fixed in https://github.com/wintercms/winter/pull/428/commits/26ecdb002e576d5321374c7ae073579054dcf5d9

**Pull request merged**